src/app/Shared/MatchExpressionEvaluator.tsx

```diff
@@ -149,7 +149,19 @@ export const MatchExpressionEvaluator: React.FunctionComponent<MatchExpressionEv
     }
     body = JSON.stringify(body, null, 2);
     body = body.substring(1, body.length - 1);
-    return body;
+    return !props.inlineHint ? (
```

Comment:
Maybe here can just continue to return the string. And use code block action here:

https://github.com/cryostatio/cryostat-web/blob/f8ae71e528a2ada04da5975e49efc223dbcceaff/src/app/Shared/MatchExpressionEvaluator.tsx#L186

Reference: https://www.patternfly.org/v4/components/code-block

Example:

![Screenshot from 2022-12-08 21-45-29](https://user-images.githubusercontent.com/68053619/206612601-260cf909-351f-4793-8426-44c50bc385bc.png)